### PR TITLE
Emit css tags

### PR DESCRIPTION
**Pull request by michaelletzgus**
More unified templates.

### CODE REVIEW

**Comment by MorrisJobke** [comment]
@michaelletzgus Could you rebase because the other PR was merged.

**Comment by michaelletzgus** [comment]
Rebased!

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

File: lib/private/legacy/template/functions.php [hunk 1]
```diff
@@ -38,6 +38,35 @@ function p($string) {
 	print(\OCP\Util::sanitizeHTML($string));
 }
 
+
+/**
+ * Prints a <link> tag for loading css
+ * @param string $href the source URL, ignored when empty
+*/
+function emit_css_tag($href, $opts) {
```
Comment (ChristophWurst) [inline lib/private/legacy/template/functions.php]:
```
"Exception: {\"Exception\":\"ArgumentCountError\",\"Message\":\"Too few arguments to function emit_css_tag(), 1 passed in \\\/nextcloud\\\/lib\\\/private\\\/legacy\\\/template\\\/functions.php on line 63 and exactly 2 expected\"
```

-> the second parameter should be optional

**Review by ChristophWurst — DISMISSED** [review]

**Comment by michaelletzgus** [comment]
Done!

**Comment by MorrisJobke** [comment]
@juliushaertl @ChristophWurst @nickvergessen Mind to re-review this one?

**Review by juliusknorr — APPROVED** [review]
Works as expected. :+1:

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**